pkg/storage/utitiltyStore.go

```diff
@@ -0,0 +1,24 @@
+package storage
+
+import (
+	_ "embed"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+)
+
+//go:embed SQL/utility/set_session_current_user.sql
+var setSessionCurrentUserSQL string
+
+// setCurrentSessionUserVariable sets the userID for
```

Comment:
Can you complete this comment? Might also be good to clarify the purpose of this function, as it's not immediately clear what we might be using this session variable for right now

Reply:
good catch! thanks for spotting it